src/main/kotlin/apply/domain/applicationform/ApplicationForm.kt

```diff
@@ -20,12 +17,9 @@ class ApplicationForm(
     var referenceUrl: String,
 
     @Embedded
-    var answers: Answers,
-
-    @Id
-    @GeneratedValue(strategy = GenerationType.IDENTITY)
-    val id: Long = 0L
-) {
+    var applicationFormAnswers: ApplicationFormAnswers,
```

Comment:
```suggestion
    var answers: ApplicationFormAnswers,
```
`ApplicationForm`내부에 있는 `answers`이니만큼 풀네임을 표시할 필요는 없지 않을까요?

Reply:
그게 좋겠네요 👍